src/cli-generate-cross-account-api-keys.ts

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env node
+
+import yargs from 'yargs-parser';
+import colors from 'colors';
+import { writeFileSync } from 'fs';
+import keyBy from 'lodash/keyBy';
+import { ScopeName, TRANSCEND_SCOPES } from '@transcend-io/privacy-types';
+
+import { logger } from './logger';
+import { generateCrossAccountApiKeys } from './api-keys';
+
+const SCOPES_BY_TITLE = keyBy(
+  Object.entries(TRANSCEND_SCOPES).map(([name, value]) => ({
+    ...value,
+    name,
+  })),
+  'title',
+);
+const SCOPE_TITLES = Object.keys(SCOPES_BY_TITLE);
+
+/**
+ * Create API keys with the same set of scopes across multiple Transcend instances
+ *
+ * Requires the username and password of user with access to multiple instances.
+ *
+ * Dev Usage:
+ * yarn ts-node ./src/cli-generate-cross-account-api-keys.ts --email=test@transcend.io --email=NE8pGp$s8Gm4Mzb5 \
+ *   --apiKeyTitle="My Test Key" --scopes="View Email Templates,View Data Map" --file=./secrets.json
+ *
+ * Standard usage:
+ * yarn tr-generate-api-keys --email=test@transcend.io --password=NE8pGp$s8Gm4Mzb5 \
+ *   --apiKeyTitle="My Test Key" --scopes="View Email Templates,View Data Map" --file=./secrets.json
+ */
+async function main(): Promise<void> {
+  // Parse command line arguments
+  const {
+    transcendUrl = 'https://api.transcend.io',
+    file,
+    email,
+    password,
```

Comment:
Just a nitpick: CLI tools that take in passwords do not recommend passing them into the command itself, as that gets saved in shell history, and usually have an option to prompt the user for a password on script startup, before any other operations take place. We may want to consider implementing support for that for added security

Reply:
yup great idea

Reply:
Yeah if it were set using a shell var and passed in like how you have it there, it would not save the resolved password, so that would be fine!